doc/source/virtual-cluster/cli.rst

```diff
@@ -0,0 +1,370 @@
+Virtual Cluster Management
+===========================
+
+.. _virtual-cluster-cli:
+
+Create a Virtual Cluster
+------------------------
+
+**Description:**
+
+Creates a new Virtual Cluster with specified parameters, allowing you to define clusters with specific replica distributions and divisibility settings.
+
+**Command:**
+
+`ray vcluster create [OPTIONS]`
+
+**Options:**
+
+.. list-table::
+    :widths: 20 16 16 16 60
+    :header-rows: 1
+
+    * - Option
+      - Type
+      - Default
+      - Required
+      - Description
+    * - --address TEXT
+      - str
+      - None
+      - NO
+      - Specifies the Ray cluster address. If not provided, the **RAY_ADDRESS** environment variable is used.
+    * - --id TEXT
+      - str
+      - N/A
+      - YES
+      - Assigns a unique identifier to the Virtual Cluster being created.
+    * - --divisible
+      - bool
+      - False
+      - NO
+      - Determines if the Virtual Cluster is divisible into smaller logical or job clusters.
+    * - --replica-sets TEXT
+      - dict
+      - N/A
+      - YES
+      - JSON-serialized dictionary defining the replica sets for the cluster (e.g., **'{"group1":2,"group2":3}'**).
+
+Usage Examples
+~~~~~~~~~~~~~~
+
+**Example:** Creating a Divisible Virtual Cluster
+
+`ray vcluster create --id myCluster --divisible --replica-sets '{"group1":2,"group2":3}'`
+
+**Output:**
+
+`Virtual cluster 'myCluster' created successfully`
+
+Update a Virtual Cluster
+------------------------
+
+**Description:**
+
+Update an existing Virtual Cluster with specified parameters.
+
+**Command:**
+
+`ray vcluster update [OPTIONS]`
+
+**Options:**
+
+.. list-table::
+    :widths: 20 16 16 16 60
+    :header-rows: 1
+
+    * - Option
+      - Type
+      - Default
+      - Required
+      - Description
+    * - --address TEXT
+      - str
+      - None
+      - NO
+      - Specifies the Ray cluster address. If not provided, the **RAY_ADDRESS** environment variable is used.
+    * - --id TEXT
+      - str
+      - N/A
+      - YES
+      - Assigns a unique identifier to the Virtual Cluster being created.
+    * - --divisible
+      - bool
+      - False
+      - NO
+      - Determines if the Virtual Cluster is divisible into smaller logical or job clusters.
+    * - --replica-sets TEXT
+      - dict
+      - N/A
+      - YES
+      - JSON-serialized dictionary defining the replica sets for the cluster (e.g., **'{"group1":2,"group2":3}'**).
```

Comment:
add following to description: Here, group1 and group2 correspond to the node type names passed via the RAY_NODE_TYPE_NAME environment variable.